packages/components/tag/src/tag-list/tag-list.tsx

```diff
@@ -18,7 +25,10 @@ const TagListItem = styled.div`
 
 const TagList = (props: TTagListProps) => {
   return (
-    <TagListContainer>
+    <TagListContainer
+      className={props.className}
```

Comment:
From my point of view, `className` is something to avoid from a design system perspective by default and only to be used with a solid reason. The goal os a design system is to create a visual representation of a brand and consumers should only be able to tailor the experience in a controlled manner so the brand can make sure its guidelines are followed. Otherwise, you could use a set of styleless components (eg: [headlessui](https://headlessui.com/), [radix](https://www.radix-ui.com/)).

That said, I totally get your point about the current state of the MC and (temporary) introducing this prop could be the best worst solution. 
Maybe don't mention it in the docs and also mark it already as depretated.